recipe/meta.yaml

```diff
@@ -89,6 +86,11 @@ test:
     - pybamm.solvers
     - pybamm.solvers.c_solvers
     - pybamm.spatial_methods
+  requires:
+    - pip
+    - python {{ python_min }}
+  commands:
+    - pip check
```

Comment:
I haven't come across this before, so I'm curious – what does this command do?

Reply:
This is used to check if any broken requirements or incompatible package versions were installed.

Reply:
I see, thanks!